src/masternode/masternode-payments.cpp

```diff
@@ -278,7 +275,7 @@ std::map<int, std::string> GetRequiredPaymentsStrings(int nStartHeight, int nEnd
         auto projection = deterministicMNManager->GetListAtChainTip().GetProjectedMNPayees(nEndHeight - nChainTipHeight);
         for (size_t i = 0; i < projection.size(); i++) {
             auto payee = projection[i];
-            int h = nChainTipHeight + 1 + i;
+            size_t h = nChainTipHeight + 1 + i;
```

Comment:
If you really want correct types there even for parameter in `mapPayments.emplace` and `GetRequiredPaymentsString` you could do something like this maybe
```suggestion
            int nOffset{0};
            for (const auto& payee : projection) {
                int h = nChainTipHeight + 1 + nOffset++;
                mapPayments.emplace(h, GetRequiredPaymentsString(h, payee));
            }
```

Reply:
Idk, that's not a great change IMO, probably makes it a bit less readable